foreman/data_refinery_foreman/surveyor/test_harmony.py

```diff
@@ -0,0 +1,299 @@
+            'E-GEOD-37912',
+            'E-GEOD-32887',
+            'E-GEOD-19314',
+            'E-GEOD-18781',
+            'E-GEOD-16538',
+            'E-GEOD-66795',
+            'E-GEOD-40568',
+            'E-MTAB-2073',
+            'E-GEOD-51092',
+            'E-GEOD-48378',
+            'E-GEOD-40611',
+            'E-GEOD-23117',
+            'E-MEXP-1883',
+            'E-GEOD-81292',
+            'E-GEOD-76886',
+            'E-GEOD-76809',
+            'E-GEOD-65405',
+            'E-GEOD-65336',
+            'E-GEOD-58095',
+            'E-MEXP-1214',
+            'E-GEOD-48149',
+            'E-MEXP-32',
+            'E-GEOD-33463',
+            'E-GEOD-32413',
+            'E-GEOD-19617',
+            'E-MTAB-1944',
+            'E-GEOD-17114',
+    ]
+
+        for accession in lots:
+            metadata = parse_sdrf("https://www.ebi.ac.uk/arrayexpress/files/" + accession + "/" + accession + ".sdrf.txt")
+            if not metadata:
+                continue
+            # No assertions, just making sure we don't barf.
+            harmonized = harmonize(metadata)
+
+    def test_sra_harmony(self):
+        """
+        Tests a specific harmonization from SRA
+        """
+        
+        metadata = SraSurveyor.gather_all_metadata("SRR1533126")
+        harmonized = harmonize([metadata])
+
+        title = 'Phosphaturic mesenchymal tumour (PMT) case 2 of NTUH'
+        self.assertTrue(title in harmonized.keys())
+        self.assertTrue('sex' in harmonized[title].keys())
+        self.assertEqual('female', harmonized[title]['sex'])
+
+        self.assertTrue('age' in harmonized[title].keys())
+        self.assertEqual(57.0, harmonized[title]['age'])
+
+        self.assertTrue('part' in harmonized[title].keys())
+        self.assertTrue('disease' in harmonized[title].keys())
+
+    @tag('slow')
+    def test_sra_lots(self):
+        """
+        Smoke tests a few SRA types
+        """
+
+        # These can be built via
+        #    https://www.ncbi.nlm.nih.gov/sra
+        # Searching for 
+        #    (human) NOT cluster_dbgap[PROP] 
+        # And then Sent To -> File -> Accession List
+        lots = [
+            'ERR188021',
+            'ERR188022',
+            'ERR205021',
+            'ERR205022',
+            'ERR205023',
+            'SRR000001', # Soft fail, bad platform
+            'ERR1737666',
+            'ERR030891',
+            'ERR030892',
+            'SRR1542948',
+            'SRR1553477',
+            'SRR1542330',
+            'SRR1538698',
+            'SRR1538760',
+            'SRR1538866',
+            'SRR1539218',
+            'SRR1797277',
+            'SRR1533126'
+        ]
+        for accession in lots:
+            try:
+                metadata = SraSurveyor.gather_all_metadata(accession)
+                harmonized = harmonize([metadata])
+                print(harmonized)
+            except UnsupportedDataTypeError as udte:
+                print(udte)
```

Comment:
It looks like these tests will be pretty noisy, did you mean to leave all these prints in here?